app/views/application/equality-monitoring/ethnic-background.html

```diff
@@ -115,15 +83,15 @@
         }
       },
       items: [{
-        text: "Asian and White"
+        text: "White and Black Caribbean"
       }, {
-        text: "Black African and White"
+        text: "White and Black African"
       }, {
-        text: "Black Caribbean and White"
+        text: "White and Asian"
       }, {
-        text: "Another Mixed background",
+        text: "Any other Mixed or Multiple ethnic background",
```

Comment:
The Census used `Any other Mixed or Multiple background` for this option.

Reply:
I took the wording from the [Equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/).

Reply:
Will stick with the wording from the GDS pattern for now, and raise this issue upstream.